Doc/c-api/structures.rst

```diff
@@ -637,24 +637,24 @@ Macro name                       C type                        Python type
    Reading a ``NULL`` pointer raises :py:exc:`AttributeError`.
 
 .. index::
-   single: T_BYTE
-   single: T_SHORT
-   single: T_INT
-   single: T_LONG
-   single: T_LONGLONG
-   single: T_UBYTE
-   single: T_USHORT
-   single: T_UINT
-   single: T_ULONG
-   single: T_ULONGULONG
-   single: T_PYSSIZET
-   single: T_FLOAT
-   single: T_DOUBLE
-   single: T_BOOL
-   single: T_CHAR
-   single: T_STRING
-   single: T_STRING_INPLACE
-   single: T_OBJECT_EX
+   single: T_BYTE (C macro)
```

Comment:
Shouldn't all of these have a `Py_` prefix?

Reply:
These are legacy names, which require including `<structmembers.h>`. There are also aliases with the `Py_` prefix.

Reply:
Okay. Again, it would be nice to have some way to identify them as legacy/deprecated/discouraged names.